README.md

```diff
@@ -151,6 +151,17 @@ which will check and use pre-built binaries if they're available for this releas
 
 to always force building the Node.js bindings from source.
 
+#### Unscoped packages
+
+Prior to v5.27.0, the `osrm` Node package was unscoped. If you are upgrading from an old package, you will need to do the following:
```

Comment:
Ideally we'd also deprecate the unscoped version

https://docs.npmjs.com/cli/v7/commands/npm-deprecate